### PR TITLE
cli: Add uvvid command and package

### PR DESCRIPTION
**Pull request by mrshu**
- Add `uvvid` command that will be created using setup.py
- Make `uvvid` look more like a package thanks to a `setup.py` file and
  a clean folder/file structure.
- Move data files into the `data/` directory.

Signed-off-by: mr.Shu mr@shu.io

### CODE REVIEW

**Comment by mrshu** [comment]
@Foxiy I would appreciate your review.

@adman if you could take a look, that would also be awesome!

File: uvvid/cli.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import click
+import cv2 as cv
+
+from .core import UVVID
+
+DEBUG = False
+
+
+@click.group()
+@click.option('--debug/--no-debug', default=True)
```
Comment (mrshu) [inline uvvid/cli.py:10]: @Foxiy @Adman note that debug is turned on by default. We might want to change that in the future.

**Comment by Adman** [comment]
@mrshu Looks nice to me. :+1:

**Comment by vidriduch** [comment]
@mrshu LGTM

**Pull request merged**